Trimble.Modus.Components/Controls/TMSegmentedControl/TMSegmentedControl.xaml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Trimble.Modus.Components.TMSegmentedControl"
+             xmlns:local="clr-namespace:Trimble.Modus.Components.Controls.Layouts"
+             xmlns:component="clr-namespace:Trimble.Modus.Components"
+             xmlns:color="clr-namespace:Trimble.Modus.Components.Styles"
+             xmlns:system="clr-namespace:System;assembly=netstandard"
+             x:Name="SegmentedControl">
+  <ContentView.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <color:Colors />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </ContentView.Resources>
+  <ContentView.Content>
+    <Frame x:Name="FrameView"
+           Padding="0"
+           IsClippedToBounds="True"
+           HeightRequest="32"
+           BorderColor="{x:StaticResource SegmentBorderColor}"
+           CornerRadius="8">
+      <Grid x:Name="TabButtonHolder"
```

Comment:
I was able to achieve this with the StackLayout itself. Can you give it a try? We can get rid of all the `ColumnDefinitions` logic. 

`<StackLayout x:Name="TabButtonHolder"
                   Orientation="Horizontal"
                   HorizontalOptions="FillAndExpand"
                   IsEnabled="True"
                   BindableLayout.ItemsSource="{Binding Source={x:Reference SegmentedControl}, Path=SegmentedItems}">
        <BindableLayout.ItemTemplate>
          <DataTemplate>
            <component:TMSegmentedItem Text="{Binding Text}"
                                       HorizontalOptions="FillAndExpand"
                                       Icon="{Binding Icon}"
                                       ShowSeparator="{Binding ShowSeparator}"
                                       IsSelected="{Binding IsSelected}"
                                       ColorTheme="{Binding ColorTheme}" />
          </DataTemplate>
        </BindableLayout.ItemTemplate>
      </StackLayout>`

Reply:
Yes, this work. Replaced it